Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.h

```diff
@@ -140,13 +139,19 @@ namespace Multiplayer
         void DumpStats(const AZ::ConsoleCommandContainer& arguments);
         //! @}
 
+        //! AzFramework::RootSpawnableNotificationBus::Handler
+        //! @{
+        void OnRootSpawnableReady([[maybe_unused]] AZ::Data::Asset<AzFramework::Spawnable> rootSpawnable, [[maybe_unused]] uint32_t generation) override;
```

Comment:
You don't need maybe_unused in the declaration.

Reply:
thank you, fixed!